src/main/java/org/fogbowcloud/app/core/IguassuController.java

```diff
@@ -119,11 +119,7 @@ public JDFJob buildJob(String jdfFilePath, User owner) throws CompilerException
 
         String externalOAuthToken = getAccessTokenByOwnerUsername(userName);
 
-        JDFJob jobBuilt = buildJobFromJDFFile(job, jdfFilePath,jobSpec, userName, externalOAuthToken);
-
-        this.createdJobs.put(jobBuilt.getId(), jobBuilt);
-
-        return job;
+        return buildJobFromJDFFile(job, jdfFilePath,jobSpec, userName, externalOAuthToken);
```

Comment:
don't we need to put it on the createdJobs anymore? why?

Reply:
This data structure is no longer used anywhere. It made sense in the blowout context that saved one thread per job, just like the map of finished tasks.

Reply:
so, why don't you remove it? It is still declared

Reply:
Done.